### PR TITLE
Update dependencies, upgrade preferences and goodbye to anko

### PR DESCRIPTION
**Pull request by robbi5**
(Currently based on the pin-dialog #54 branch, will be rebased when that got merged) 

This PR updates gradle, kotlin and some dependencies, introduces flipper in pretixPRINT too, migrates our settings/preferences to androidx, the styling to Material3 and finally completely removes anko \o/
I've tried to split the parts into single commits for easier review.

TODO:
- [x] Update about.html with dependency licenses

### CODE REVIEW

File: pretixprint/app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -1,7 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:tools="http://schemas.android.com/tools"
-    package="eu.pretix.pretixprint">
```
Comment (raphaelm) [inline pretixprint/app/src/main/AndroidManifest.xml:4]: Is that removal on purpose?

Comment (robbi5) [reply pretixprint/app/src/main/AndroidManifest.xml:4]: yep, Android Studio removed that automatically, it is now replaced by the `applicationId` in build.gradle
